raaghu-components/src/rds-comp-contact-us/rds-comp-contact-us.tsx

```diff
@@ -14,8 +14,7 @@ const RdsCompContactUs = (props: RdsCompContactUsProps) => {
     const [error2, setError2] = useState("");
```

Comment:
@harshitsaini-waiin give proper name instead error1, error2 and error3 give it as per field name